fitness-tracker/src/app/logs/log.service.ts

```diff
@@ -28,6 +28,13 @@ export class LogService{
         }));
     }
 
+    getUserLogs(id:string):Observable<ILog[]>{
+        return this.httpClientService.get<ILog[]>(`${path}/GetUserLogs/${id}`)
+        .pipe(map( (logs:ILog[]) =>{
+            return logs.map( log => createLog(log))
```

Comment:
remove the function export stuff. use models here and expose Interfaces to consumers of the service.